ENGAGEHF/Dashboard/Vitals/Views/VitalsCard.swift

```diff
@@ -0,0 +1,109 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+import SwiftUI
+
+
+struct VitalsCard: View {
+    @ScaledMetric private var quantityTextSize: CGFloat = 20
```

Comment:
I think we could benefit from a larger font size here:
```suggestion
    @ScaledMetric private var quantityTextSize: CGFloat = 40
```